Docs/Configuration.tex

```diff
@@ -2400,6 +2400,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note}: This option should be avoided whenever possible. Modern firmware
   usually have compatible AHCI controllers.
+  
+  \item
+  \texttt{ExternalDiskIconsNVME}\\
```

Comment:
I suggest NVMe instead of NVME. In both the quirk name and all the occurrences of the name. This is the correct spelling.

Reply:
Done